### PR TITLE
Added bbtools 38.86

### PR DESCRIPTION
**Pull request by AbigailShockey**

### CODE REVIEW

**Comment by kapsakcj** [comment]
Hi @AbigailShockey thanks for the PR. Could you please add the new bbtools version to the README in another commit to your forked repo?

Also - ignore Travis. A lot of the travis builds/tests are failing due to the tests taking exceedingly long (has nothing to do with your PR). We may need to re-think our testing strategy at some point.

File: bbtools/38.86/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,36 @@
+FROM ubuntu:xenial
+
+LABEL base.image="ubuntu:xenial"
+LABEL version="1"
```
Comment (kapsakcj) [inline bbtools/38.86/Dockerfile:4]: could you please adjust this line to `LABEL dockerfile.version="1"`

File: bbtools/38.86/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+RUN conda config --add channels defaults && \
+  conda config --add channels bioconda && \
+  conda config --add channels conda-forge && \
+  conda install sambamba
+
+RUN wget https://sourceforge.net/projects/bbmap/files/BBMap_38.86.tar.gz && \
+  tar -xzf BBMap_38.86.tar.gz && \
+  rm BBMap_38.86.tar.gz
+
+ENV PATH="${PATH}:/bbmap"
```
Comment (kapsakcj) [inline bbtools/38.86/Dockerfile:35]: Might be worth adding in LC_ALL here too, for singularity compatibility
```Dockerfile
ENV PATH="${PATH}:/bbmap"\
 LC_ALL=C
```

File: bbtools/38.86/Dockerfile (hunk 1 above)
Comment (AbigailShockey) [reply bbtools/38.86/Dockerfile]: I've added this

File: bbtools/38.86/Dockerfile (hunk 2 above)
Comment (AbigailShockey) [reply bbtools/38.86/Dockerfile]: And this

File: bbtools/38.86/Dockerfile (hunk 1 above)
Comment (kapsakcj) [reply bbtools/38.86/Dockerfile]: ty!

File: bbtools/38.86/Dockerfile (hunk 2 above)
Comment (kapsakcj) [reply bbtools/38.86/Dockerfile]: ty!

**Comment by kapsakcj** [comment]
Looks good to go! Thanks for the PR and making my requested changes!

**Comment by kapsakcj** [comment]
building on dockerhub now

**Comment by kapsakcj** [comment]
^ I was referring to bbtools 38.86 (also switching staphb/bbtools:latest to 38.86). I'm also re-building staphb/cluster-report-env:1.0.

Let me know if you need anything else re-built on dockerhub

**Pull request merged**